api/src/config.rs

```diff
@@ -83,7 +83,10 @@ impl ConfigV2 {
     pub fn from_file<P: AsRef<Path>>(path: P) -> Result<Self> {
         let md = fs::metadata(path.as_ref())?;
         if md.len() > 0x100000 {
-            return Err(eother!("configuration file size is too big"));
```

Comment:
It's a limitation of rust, 
```
error: macro-expanded `macro_export` macros from the current crate cannot be referred to by absolute paths
  --> api/src/config.rs:17:5
   |
17 | use crate::eother;
   |     ^^^^^^^^^^^^^
   |
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #52234 <https://github.com/rust-lang/rust/issues/52234>
note: the macro is defined here
  --> api/src/error.rs:33:9
   |
33 | /         macro_rules! $fn {
34 | |             () => {
35 | |                 std::io::Error::new($err.kind(), format!("{}: {}:{}", $err, file!(), line!()))
36 | |             };
...  |
39 | |             };
40 | |         }
   | |_________^
...
82 |   define_error_macro!(eother, std::io::Error::new(std::io::ErrorKind::Other, ""));
   |   ------------------------------------------------------------------------------- in this macro invocation
   = note: `#[deny(macro_expanded_macro_exports_accessed_by_absolute_paths)]` on by default
   = note: this error originates in the macro `define_error_macro` (in Nightly builds, run with -Z macro-backtrace for more info)

error: macro-expanded `macro_export` macros from the current crate cannot be referred to by absolute paths
    --> api/src/config.rs:1258:17
     |
1258 |                 crate::eother!("test");
     |                 ^^^^^^^^^^^^^
     |
     = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
     = note: for more information, see issue #52234 <https://github.com/rust-lang/rust/issues/52234>
note: the macro is defined here
    --> api/src/error.rs:33:9
     |
33   | /         macro_rules! $fn {
34   | |             () => {
35   | |                 std::io::Error::new($err.kind(), format!("{}: {}:{}", $err, file!(), line!()))
36   | |             };
...    |
39   | |             };
40   | |         }
     | |_________^
...
82   |   define_error_macro!(eother, std::io::Error::new(std::io::ErrorKind::Other, ""));
     |   ------------------------------------------------------------------------------- in this macro invocation
     = note: this error originates in the macro `define_error_macro` (in Nightly builds, run with -Z macro-backtrace for more info)

```

Reply:
OK, I get it.